src/features/selectedPost.ts

```diff
@@ -0,0 +1,27 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { Post } from '../types/Post';
+
+export interface SelectedPostState {
+  selectPost: Post | null;
+}
+
+const initialState: SelectedPostState = {
+  selectPost: null,
+};
+
+export const selectedPostSlice = createSlice({
+  name: 'selectPost',
+  initialState,
+  reducers: {
+    currentPost: (state, action: PayloadAction<Post | null>) => {
```

Comment:
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/79990025-f67f-4d18-90d3-9ef66a2ba9ee)

it was about this action name
```suggestion
    selectPost: (state, action: PayloadAction<Post | null>) => {
```